### DISCUSSION TITLE
cli unable to login

### DISCUSSION THREAD

**Discussion by gpldecha** [cvat-ai/cvat#4419] [Q&A]
Hi,

I have been trying to use the cli tool.  I have not been able to login via cli:

`/cli.py --auth USER:PASS --server-host IP --server-port PORT ls`

I get the following error:

`requests.exceptions.HTTPError: 404 Client Error: Not Found for url: http://IP:PORT/api/auth/login`

When I enter into my browser:

`http://IP:PORT/auth/login` It takes me to the login page (dropping the api).

Any help would be appreciated.

Thanks

**Comment by azhavoro** [comment]
Hi,

most likely you have cli and CVAT instance version mismatch, please checkout to cli version that corresponds your deployed version or upgrade your CVAT instance
